libbeat/docs/newbeat.asciidoc

```diff
@@ -59,73 +59,141 @@ must contain a `@timestamp` field and a `type` field. Beyond
 that, events can contain any additional fields, and they can be created as often
 as necessary.
 
-The following example shows an event object in Topbeat:
+The following example shows an event object in Lsbeat:
 
 [source,json]
 ----------------------------------------------------------------------
 {
-  count":1,
-  "proc.cpu":{
-    "user":20,
-    "percent":0.983284169124877,
-    "system":0,
-    "total":20,
-    "start_time":"20:20"
+  "@timestamp": "2016-07-13T21:33:58.355Z",
+  "beat": {
+    "hostname": "mar.local",
+    "name": "mar.local"
   },
-  "proc.mem":{
-    "size":333772,
-    "rss":6316,
-    "share":2996
+  "directory": false,
+  "filename": "winlogbeat.yml",
+  "filesize": 2895,
+  "modtime": "2016-07-13T20:56:21.000Z",
+  "path": "./vendor/github.com/elastic/beats/winlogbeat/winlogbeat.yml",
+  "type": "lsbeat"
+}
+
+{
+  "@timestamp": "2016-07-13T21:33:58.354Z",
+  "beat": {
+    "hostname": "mar.local",
+    "name": "mar.local"
   },
-  "proc.name":"topbeat",
-  "proc.pid":13954,
-  "proc.ppid":10027,
-  "proc.state":"sleeping",
-  "shipper":"vagrant-ubuntu-trusty-64",
-  "@timestamp":"2015-08-06T20:20:34.089Z",
-  "type":"proc"
+  "directory": true,
+  "filename": "system",
+  "filesize": 238,
+  "modtime": "2016-07-13T20:56:21.000Z",
+  "path": "./vendor/github.com/elastic/beats/winlogbeat/tests/system",
+  "type": "lsbeat"
 }
 ----------------------------------------------------------------------
 
 Now that you have the big picture, let's dig into the code.
 
+=== Generate your Beat
+
+To generate your own beat we use the `beat-generator`. First you must install
+https://github.com/audreyr/cookiecutter[cookiecutter]. Check out the
+http://cookiecutter.readthedocs.io/en/latest/installation.html[installation
+guide]. After having installed cookiecutter, we must decide on a name for our beat. 
+The name must be one word all lowercase. In our example is lsbeat.
+
+To create the Beat skeleton, you should download Beats generator package,available in the beats repository. 
+Once you installed GoLang, you can download the Beats generator package using
+go get command. Once you run the command, all source files will be downloaded under the $GOPATH/src path.
```

Comment:
Maybe reference the generator's readme: https://github.com/elastic/beats/tree/master/generate/beat